inductiva/__init__.py

```diff
@@ -23,7 +23,14 @@
 
 api_url = os.environ.get("INDUCTIVA_API_URL", "http://api.inductiva.ai")
 output_dir = os.environ.get("INDUCTIVA_OUTPUT_DIR", "inductiva_output")
-api_key = os.environ.get("INDUCTIVA_API_KEY")
+
+credentials_file_path = os.getenv("INDUCTIVA_API_CREDENTIALS")
```

Comment:
Yes, it's a good idea. Speaking about defaults, [here's how Google Cloud implements default files](https://cloud.google.com/docs/authentication/application-default-credentials#personal), once we have a console up, I think this will be the right place to define the default credentials directory where in case of web login. This also addresses @sarmento's comments made yesterday.